### DISCUSSION TITLE
記事本体の左側のデッドスペースに関して

### DISCUSSION THREAD

**Discussion by m-tmatma** [increments/qiita-discussions#108] [要望・アイデア・不具合]
記事の表示画面で、記事本体の左側に関して、大きなディスプレイで表示したとしても
何も表示されない空白の領域になっているのは非常に使いにくいです。

記事左側には LGTM 等のアイコンが縦に並んでいるだけで、デッドスペースになっています。

記事の中で、コマンドやログなどを記載するときに、記事本体の表示幅が狭いせいで、見にくくなっています。

記事左側の領域を有効活用するようなレイアウトにしてほしいです。

**Comment by kskwtnk** [comment]
@m-tmatma 

ご意見ありがとうございます！

こちらは、タイトルこそ「記事本体の左側のデッドスペース」ですが、内容として「記事の表示の見やすさの問題」なのかと思い、その観点で回答します 🙏 

人間が読みやすい行の長さは決まっており、40文字弱と言われています
Qiitaは現状でもむしろ若干長い幅になっているため

> 記事本体の表示幅が狭いせいで、見にくくなっています。

上記を解消するために記事のエリアの幅を広げる判断はしない予定です

（ただしコードブロックの改行される・されないといったスタイリングはあまり見やすくないので、ここを個別で改善することはあると思います）

ご理解いただけますと幸いです！

**Reply by m-tmatma** [reply]
@xrxoxcxox 

> 内容として「記事の表示の見やすさの問題」なのかと思い、その観点で回答します 

はい、その認識です。
「記事本体の左側のデッドスペース」が余っているので、見やすさを改善するための手段として書きました。

> 人間が読みやすい行の長さは決まっており、40文字弱と言われています

これは文章だけの話であって、表、画像、あるいは箇条書きでインデントを含む場合などは
別なのではないかと思います。

> （ただしコードブロックの改行される・されないといったスタイリングはあまり見やすくないので、ここを個別で改善することはあると思います）

事実上記で指摘されているように、コードブロックが改行されている場合は非常に見にくいです。

表や画像含めて、いろんな表現方法があるので、行の長さが 40文字という原則はあると思いますが、
全体として見やすくなるのはどういたらいいかという原点に立ち戻って考えていただいて
最適な表示方法を検討いただけるとありがたいです。

**Reply by kskwtnk** [reply]
@m-tmatma 
返信が漏れており、申し訳ありません！

> 全体として見やすくなるのはどういたらいいかという原点に立ち戻って考えていただいて
> 最適な表示方法を検討いただけるとありがたいです。

はい、こちら模索していきます！
記事左側のスペースを……という話とは変わってしまうかもしれませんが、文章やコード、画像などどの表現でも見やすくなるように改善していきます 🙏
